src/components/Input/RadioButton/RadioButton.js

```diff
@@ -81,8 +91,14 @@ const RadioButton = ({
                 )}
                 {...props}
                 isFocused={focused === index}
+                description={description}
               />
-              <RadioText>{children}</RadioText>
+              <RadioText description={description}>
+                {children}
```

Comment:
Just wondering, if you it was tried to pass already styled text (with secondary label if that's needed) to RadioText component with children, instead of customizing lib? What do you think?